components/dashboard/src/components/WidePageWithSubmenu.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import classNames from "classnames";
+import { Separator } from "./Separator";
+import { cn } from "@podkit/lib/cn";
+import { SubmenuItem, SubmenuItemProps } from "./PageWithSubMenu";
+
+export interface PageWithSubMenuProps {
+    subMenu: SubmenuItemProps[];
+    children: React.ReactNode;
+}
+
+export function WidePageWithSubMenu(p: PageWithSubMenuProps) {
+    return (
+        <div className="w-full">
+            <div className={cn("app-container flex flex-col md:flex-row")}>
+                {/* TODO: extract into SubMenu component and show scrolling indicators */}
+                <div>
```

Comment:
This could potentially be a <nav>. Need to make sure if it's semantically alright to have multiple of these per page (mindful of the top nav we employ), but a div it probably not the right call here.

Reply:
It is now